client/packages/sdk/src/price-estimation/index.ts

```diff
@@ -152,13 +141,13 @@ export async function estimateBidAmount<T extends EstimateEthActionParams | Spee
  * @returns The maximum reward estimate
 */
 
-export async function estimateMaxReward<T extends EstimateEthActionParams | SpeedMode>({
-  action, asset: baseAsset, target, targetAmount, targetAsset, overSpendPercent = 0.5, args
+export async function estimateMaxReward<T extends EstimateParams>({
+  action, baseAsset, target, targetAmount, targetAsset, overSpendPercent = 0.5, args
 }: EstimateMaxReward<T>): Promise<MaxRewardEstimation> {
-  const targetNativeAsset = mapTargetToNativeAsset(target);
+  const targetNativeAsset = getTargetNativeAsset(target);
```

Comment:
you can get this via XDNS also

Reply:
In our approach to price estimation within this module, I've opted to avoid RPC calls and keep everything localized. To that end, I've implemented a method `NativeAssetMap.setFor(target, symbol)`.

If you ever need to change the default asset symbols, you can do so by:

1. Making an XDNS call for the target you want to change.
2. Calling `NativeAssetMap.setFor(target, symbol)` with the updated symbol.

This way, we can keep everything within this module and have a more streamlined process.